src/jdk.httpserver/share/classes/com/sun/net/httpserver/HttpExchange.java

```diff
@@ -233,22 +233,28 @@ protected HttpExchange() {
     public abstract String getProtocol();
 
     /**
-     * {@link Filter} modules may store arbitrary objects with {@code HttpExchange}
-     * instances as an out-of-band communication mechanism. Other filters
+     * {@return the attribute's value from this exchange's
```

Comment:
It looks a bit odd in the generated doc that this statement appears as the summary and is then repeated verbatim in the @returns. Could the first sentence be a shorter summary?

Also, since we are clarifying the spec here, could a small test be added?

Reply:
Done - I've updated the `@return` to split it into separate sentences and also updated an existing test to include the testing for `HttpExchange.getAttribute()` and `setAttribute()`. The test continues to pass. I'll run it through our CI too.